docs/architecture/cloud-native/identity-server.md

```diff
@@ -0,0 +1,101 @@
+---
+title: IdentityServer
```

Comment:
are there any other keywords that you can add to these articles with very short titles?

Reply:
We could say "for cloud native applications" if you like.

Reply:
Works for me. Whatever if you feel will help customers find this content.